snap/snapenv/snapenv.go

```diff
@@ -93,16 +93,17 @@ func snapEnv(info *snap.Info) map[string]string {
 // used by so many other modules, we run into circular dependencies if it's
 // somewhere more reasonable like the snappy module.
 func basicEnv(info *snap.Info) map[string]string {
+	// TODO parallel-install: use of proper instance/store name
 	return map[string]string{
 		// This uses CoreSnapMountDir because the computed environment
 		// variables are conveyed to the started application process which
 		// shall *either* execute with the new mount namespace where snaps are
 		// always mounted on /snap OR it is a classically confined snap where
 		// /snap is a part of the distribution package.
-		"SNAP":          filepath.Join(dirs.CoreSnapMountDir, info.Name(), info.Revision.String()),
+		"SNAP":          filepath.Join(dirs.CoreSnapMountDir, info.InstanceName(), info.Revision.String()),
 		"SNAP_COMMON":   info.CommonDataDir(),
 		"SNAP_DATA":     info.DataDir(),
-		"SNAP_NAME":     info.Name(),
+		"SNAP_NAME":     info.InstanceName(),
```

Comment:
Can you please add an explicit TODO here as well.

Reply:
There is one just above